cmd/lk/sip.go

```diff
@@ -130,10 +129,18 @@ var (
 							ArgsUsage: RequestDesc[livekit.CreateSIPParticipantRequest](),
 						},
 						{
-							Name:      "transfer",
-							Usage:     "Transfer a SIP Participant",
-							Action:    transferSIPParticipant,
-							ArgsUsage: RequestDesc[livekit.TransferSIPParticipantRequest](),
+							Name:   "transfer",
+							Usage:  "Transfer a SIP Participant",
+							Action: transferSIPParticipant,
+							Flags: []cli.Flag{
+								roomFlag,
+								identityFlag,
+								&cli.StringFlag{
+									Name:     "to",
+									Required: true,
+									Usage:    "`SIP URL` to transfer the call to. Use 'tel:<phone number>' to transfer to a phone",
```

Comment:
Is the `tel:` prefix a part of SIP spec? Did I miss the parsing part in the `REFER` PR?

Reply:
This is supported and documented by Twilio. It's unclear if other providers support it as well. We can adjust the documentation as needed.